applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp

```diff
@@ -353,83 +340,71 @@ namespace Kratos
 		KRATOS_TRY;
 		//calculate orthogonal load vector
 		const double numerical_limit = std::numeric_limits<double>::epsilon();
-		Vector GeometricOrientation = ZeroVector(3);
-		GeometricOrientation[0] = this->GetGeometry()[1].X()
+		Vector geometric_orientation = ZeroVector(3);

```

Comment:
I think there is an utility in the core that does exactly this orientation. Can u use that instead? it is just to centralize